packages/explat-react-client/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+	"compilerOptions": {
+		"target": "ES5",
+		"lib": [ "DOM", "ESNext" ],
```

Comment:
Let's only add these `lib` entries when they're necessary (to avoid accidentally including them when we don't need them.